### ISSUE TITLE
Projects list params badly taken into account

### ISSUE THREAD

**Issue by ronangaillard** [monkeyWzr/hugo-theme-cactus#20]
If you have 

```
[params]
  showProjectsList = false
```

in your config file, but you also have a `data/projects.json` in your project, the main page will display the projects anyway.

I wanted to fix it by changing the `index.htm`l template with

```go
{{ $showProjectsList := false }}
  {{ if (isset .Site.Params "showProjectsList") }}
    {{ $showProjectsList = .Site.Params.showProjectsList }}
  {{ else if .Site.Data.projects }}
    {{ $showProjectsList = true }}
  {{ end }}
```

but it still does not work (projects are still shown), I'm not a go template expert though

**Comment by monkeyWzr** [comment]
@ronangaillard hey there, so sorry for the delay and really appreciate your help!

We should use lower case when checking the key by isset. Hugo stores all site-level configuration keys as lower case.
[https://gohugo.io/functions/isset/](https://gohugo.io/functions/isset/)

`isset .Site.Params "showprojectslist"` will work.

Would you like to make a pr on that?
(the logic I wrote was totally a mess. Thanks again!)

**Comment by ronangaillard** [comment]
I'll try to make a PR soon yes :)

**Comment by monkeyWzr** [comment]
@ronangaillard  Hi there!
This has been fixed in #77.

Best regards!
